llarp/config/config.cpp

```diff
@@ -531,6 +531,18 @@ namespace llarp
           m_ExitAuths.emplace(exit, auth);
         });
 
+    conf.defineOption<bool>(
+        "network",
+        "route-poker",
```

Comment:
A better user-facing name would be "add-routes"; `route-poker` is more of an internal implementation detail name.

Reply:
Or `"auto-routing"`